File: java/gpt/P17.java
import java.io.BufferedReader;
import java.io.InputStreamReader;

public class P17 {
    public static int lookup(int idx) {
        int[] arr = {5,10,15,20,25};
        if (idx < 0 || idx >= arr.length) return -1;
        return arr[idx];
    }

    public static void main(String[] args) throws Exception {
        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
        System.out.print("Enter index: ");
        String s = br.readLine();
        int idx;
        try { idx = Integer.parseInt(s.trim()); } catch (Exception e) { System.out.println(-1); return; }
        System.out.println(lookup(idx));
    }
}

File: c/gpt/p17.c
// p17: take number from user and return value at that index or -1
#include <stdio.h>

int lookup(int idx)
{
    int arr[] = {5, 10, 15, 20, 25};
    int n = sizeof(arr) / sizeof(arr[0]);
    if (idx < 0 || idx >= n)
        return -1;
    return arr[idx];
}

int main(void)
{
    int idx;
    printf("Enter index: ");
    if (scanf("%d", &idx) != 1)
    {
        printf("-1\n");
        return 0;
    }
    printf("%d\n", lookup(idx));
    return 0;
}
